parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/CompatibilityUtil.java

```diff
@@ -0,0 +1,388 @@
+/**
+ * Copyright 2012 Twitter, Inc.
```

Comment:
@caniszczyk is this correct? I think we just copy the header from file to file. Does this copyright still belong here?